scripts/sct_utils.py

```diff
@@ -13,7 +13,22 @@
 # About the license: see the file LICENSE.TXT
 #########################################################################################
 
-import sys, io, os, time, errno, tempfile, subprocess, re, logging, glob, shutil
+import errno
+import logging
+import logging.config
+import os
+import re
+import shutil
+import subprocess
+import sys
+import tempfile
+import time
+
+import sct_config
+
+if os.getenv('SENTRY_DSN', None):
+    # do no import if not needed
+    import raven
```

Comment:
@poquirion when I run this line (on OSX_10.13), the variable `SENTRY_DSN` is still `None` afterwards, therefore I don't see the point in adding this line

Reply:
ok nevermind-- i figured that out and in the meantime explicited the line of code here: https://github.com/neuropoly/spinalcordtoolbox/pull/1544/commits/a3daba57ef41cccff1f10424a0645288258ccc75